app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"></script>
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.8/css/solid.css">
+  <script src="https://use.fontawesome.com/releases/v5.0.7/js/all.js"></script>
+  <link rel="stylesheet" type="text/css" href="sign_in_style.css">
+
+</head>
+<body>
+
+<div class="modal-dialog text-center">
+  <div class="col-sm-8 main-section">
+    <div class="modal-content">
+      <div class="col-12 user-img">
+        <!-- Get picture from user database -->
+        <%= image_tag("user_avatar.png") %>
+      </div>
+
+      <div class="col-12">
+
+        <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
```

Comment:
Ab hier beginnt der Template-Code. Das ganze drum herum, das ganze HTML ist nicht notwendig in Rails.